### ISSUE TITLE
[Whitelist] INSIDE (304430)

### ISSUE THREAD

**Issue by bscubed** [ValveSoftware/Proton#429]
[Steam System Information](https://gist.github.com/bscubed/403c4711122fa562a5352cea7516d7d5)

![screenshot from 2018-08-24 14-54-19](https://user-images.githubusercontent.com/1623826/44602458-b012c100-a7ad-11e8-9f77-8fab3433736f.png)

INSIDE runs flawlessly from start to finish.

**Comment by swick** [comment]
x360 controller is not working, everything else is fine.

    OpenGL renderer string: AMD Radeon(TM) HD 8800 Series (PITCAIRN, DRM 3.26.0, 4.18.0, LLVM 6.0.1)
    OpenGL core profile version string: 4.5 (Core Profile) Mesa 18.1.6

**Comment by ghost** [comment]
Finished the game without any issue. After that though, I loaded through levels a few times, trying to find some secrets I missed during the main play, and the audio became crackling suddenly. Then, after minimizing the game a few times it finally crashed. I don't know if it's the game's fault or if it might do that on Windows as well.
I'd say it's perfectly playable otherwise, changing resolution and fullscreen mode works as it should.

**Comment by toby3d** [comment]
Game works fine, as and my Steam Controller (but without vibration)

**Comment by pedrofleck** [comment]
Does anyone knows how to fix Xbox 360 wireless controller? It keeps sending the "left" command.

**Comment by Mondei1** [comment]
Same here. Everything runs fine with constant 60fps from start to end. And my Xbox One controller works fine (without vibration but this is a driver issue).
In my mind, it can be marked as compatible because it didn't mind one single issue with it :smile:  

**Specs**
Proton: 4.16-5 Beta
OS: Manjaro Linux 
KDE-Plasma-Version: 5.14.4
Qt-Version: 5.11.2
KDE-Framework-Version: 5.52.0
Kernel: 4.19.8-2-MANJARO
Arch: 64-bit
CPU: 4 × Intel® Core™ i5-7600K CPU @ 3.80GHz
RAM: 15,6 GiB

**Comment by MonkeySon** [comment]
![system_info](https://user-images.githubusercontent.com/10551896/52729317-37046180-2fb9-11e9-9f14-826045c64d49.png)

![system_info_graphics](https://user-images.githubusercontent.com/10551896/52729207-0a504a00-2fb9-11e9-8f56-f1d94553a3d2.png)

I am sorry to say but it does not even pass the loading screen on my configuration.
Either the game just closes (if i use PROTON_NO_ESYNC=1 and/or PROTON_NO_D3D11=1) or it just freezes (no option used).

Proton Version: 3.16-6 Beta

**Comment by kisak-valve** [comment]
Hello @MonkeySon, please verify any vulkan application like `vulkaninfo` from `vulkan-tools` works. Also, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by MonkeySon** [comment]
Hi @kisak-valve ,
I am a little bit embarrassed right now because the game now starts flawless without any options or changes. My guess is that the game just took really long time to load on the first start and I was too impatient.

Sorry for that, my bad!

**Comment by glubsy** [comment]
I can confirm this game ran perfectly well, with constant 60fps, no hitching, zero problem from start to finish, all achievements unlocked. 
Wired xbox360 controller works perfectly well too.

Using DXVK, and Proton 4.11-3. 

I would vote yes for a white list. Thanks everyone!

[My specs](https://gist.github.com/glubsy/8930de18e83adb497905097091e9789c/0fb5fbe55a58ec582be86dccd0251dce5be8b956)

**Comment by libcg** [comment]
It's crashing at launch with Proton 4.11-9. However it works perfectly with Proton 4.2-9.

Crash log: [proton_411_9_crash.txt](https://github.com/ValveSoftware/Proton/files/3909200/proton_411_9_crash.txt)

**Comment by kisak-valve** [comment]
Hello @libcg, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Some lines of interest from the log:
`Crash in winutils::ProcessInternalCrash(PEXCEPTION_POINTERS pExInfo)wine: Unhandled page fault on read access to 0x00000010 at address 0x7fa3dbd50d1d (thread 003c), starting debugger...`
`Backtrace:
=>0 0x00007fa3dbd50d1d _ZNK4llvm19XCoreTargetLowering28getRegForInlineAsmConstraintEPKNS_18TargetRegisterInfoENS_9StringRefENS_3MVTE+0xffffffffffffffff() in libllvm-10git.so (0x00007fa3a402ae30)
  1 0x00007fa3de3a95e6 _ZTSN4llvm13LiveVariablesE+0x4b35() in libllvm-10git.so (0x00007fa3a402ae30)`
`0x00007fa3dbd50d1d _ZNK4llvm19XCoreTargetLowering28getRegForInlineAsmConstraintEPKNS_18TargetRegisterInfoENS_9StringRefENS_3MVTE+0xffffffffffffffff in libllvm-10git.so: movq	0x0000000000000010(%rsi),%rbp`
There's no chance that Proton is directly using libllvm-10git.so. Instead, at first glance this looks like DXVK exposing a mesa/radv + llvm 10 driver issue.

**Comment by libcg** [comment]
You're right, I missed that part. I can confirm that it works fine with `RADV_PERFTEST=aco` or mesa 19.2.6.

[steam-304430.log](https://github.com/ValveSoftware/Proton/files/3914311/steam-304430.log)
https://gist.github.com/libcg/73a4e44d5828b42d5e2bee7c91141731

EDIT: In case it's unclear from the logs, I have mesa 19.2 for radeonsi and mesa-git for RADV installed side by side.

**Comment by libcg** [comment]
Opened a mesa issue here: https://gitlab.freedesktop.org/mesa/mesa/issues/2170

**Comment by libcg** [comment]
@kisak-valve the issue is gone after updating to the latest llvm-git and mesa-git.
